modules/component-store/src/component-store.ts

```diff
@@ -156,6 +156,29 @@ export class ComponentStore<T extends object> implements OnDestroy {
     }
   }
 
+  /**
+   * Patches the state with provided partial state.
+   *
+   * @param partialStateOrUpdaterFn a partial state or a partial updater
+   * function that accepts the state and returns the partial state.
+   * @throws Error if the state is not initialized.
+   */
+  patchState(
+    partialStateOrUpdaterFn: Partial<T> | ((state: T) => Partial<T>)
+  ): void {
+    if (!this.isInitialized) {
```

Comment:
updater does this check, so it's not needed here.

Reply:
done